### DISCUSSION TITLE
Ardunio LoRa SX1262 Module Examples Learning Materials?

### DISCUSSION THREAD

**Discussion by kkkreset** [earlephilhower/arduino-pico#1622] [Q&A]
I started to learn Arduion, but I encountered obstacles in using it. There were no suitable examples of the module.

**Comment by maxgerhardt** [comment]
The classiscal https://github.com/matthijskooijman/arduino-lmic libary now points at for https://github.com/LacunaSpace/basicmac for a library with SX1262 support. This library should only need the most basic SPI library to function and thus should work with this core no problems. Have you tried this?

Alternatively, tried https://github.com/beegee-tokyo/SX126x-Arduino/tree/master/examples/LoRaWan? That's a port of https://github.com/Lora-net/LoRaMac-node/ for Arduino.

Strictly speaking, this is not an issue of this core. Maybe a [discussion](https://github.com/earlephilhower/arduino-pico/discussions) topic.

Edit: And just to be sure, you are using a Raspberry Pi Pico (based) board and a Semtech SX1262 module, which exposes its SPI (and DIO) interface, yeah?

**Reply by kkkreset** [reply]
Ok, I will try to read and debug. Thank you for the information.
